flang/lib/Optimizer/Support/InternalNames.cpp

```diff
@@ -205,6 +205,15 @@ fir::NameUniquer::doVariable(llvm::ArrayRef<llvm::StringRef> modules,
   return result.append(toLower(name));
 }
 
+std::string
+fir::NameUniquer::doNamelistGroup(llvm::ArrayRef<llvm::StringRef> modules,
+                                  llvm::Optional<llvm::StringRef> host,
+                                  llvm::StringRef name) {
+  std::string result = prefix();
+  result.append(doModulesHost(modules, host)).append("NG");
```

Comment:
Just use one capital letter here to be consistent with all other cases of unique names,

Reply:
It looks like there are other 2-letter cases in this file, but I can change to a single letter if that is preferred.  Any suggestions?  (Hmm, why "E" instead of "V" for variable?  "E" -> entity??)

```
doModules - "M" "S"
doModulesHost - "F"
doKind - "K" "N"
doCommonBlock - "B"
doBlockData - "L"
doConstant - "EC"
doDispatchTable - "DT"
doGenerated - "Q"
doIntrinsicTypeDescriptor - "C"
doProcedure - "P"
doType - "T"
doTypeDescriptor - "CT"
doVariable - "E"

doNamelistGroup - "NG" ??
```

Reply:
I'll switch to "G".  This choice is user-visible in object code, but can be changed at least up to the time of a first official release.